### ISSUE TITLE
Symbols API: Request: Make it easier to detect measureable type definitions

### ISSUE THREAD

**Issue by alfonsogarciacaro** [dotnet/fsharp#5992]
If I get the typed AST of `let value = 0.7833263478179128134089M`, the type of `value` becomes "Microsoft.FSharp.Core.decimal\`1", whose `.AbbreviatedType` is "System.Decimal". But if I check `.IsAbbreviation` this evals to false, which makes it difficult to know whether I should check the abbreviation or not.

**Comment by realvictorprm** [comment]
Definitely a bug.
@alfonsogarciacaro Does this affect Fable anywhere critical?

**Comment by alfonsogarciacaro** [comment]
Yes. These two statements print different values in Fable but not in dotnet:

```fsharp
let value = 0.5M
value.GetType().FullName |> printfn "%s" // Microsoft.FSharp.Core.decimal`1
typeof<decimal>.FullName |> printfn "%s" // System.Decimal
```

But I've [found a workaround](https://github.com/fable-compiler/Fable/commit/4229742047f4aca43d5f26d20b9e7873f4a9c160#diff-a3efa954943b8e12ef2ab5c974d65b5f) for now, not sure how correct it is.

**Comment by alfonsogarciacaro** [comment]
Recently we've found the problem is aggravated with measure annotated abbreviations outside FSharp.Core, like [the ones in FSharp.UMX](https://github.com/fsprojects/FSharp.UMX/blob/408743115a92094ac29e3fc4ab585ace4ef3bb96/src/FSharp.UMX.fs#L10-L17). In that case I haven't found a way to access the abbreviated type, the work around linked in the previous comment doesn't work, not sure why (though I guess it's because the measure annotated abbreviations are hard-coded somewhere in the F# compiler).

**Comment by dsyme** [comment]
You should look for `MeasureAnnotatedAbbreviation` attribute (MeasureAnnotatedAbbreviationAttribute) in the attributes of the type definition.  In this case, the type represents a new type definition that is a measure-annotated version of the existing type definition.  It acts like a new type, and is not automatically expanded to the underlying type except when instantiated with the unit-of-measure "1"
